### DISCUSSION TITLE
Failed to find current location

### DISCUSSION THREAD

**Discussion by zhmstar0310** [breezy-weather/breezy-weather#853] [Q&A]
it's difficult to update current location, I choose location service is Android, did I do wrong settings, thanks
![image](https://github.com/breezy-weather/breezy-weather/assets/873345/53081341-b1f1-4c51-a695-1326dafafd9f)

**Comment by papjul** [comment]
Can you try settling your location in a different maps application and try refreshing weather after that?

Do you use Google Play Services (if you have Google Play Store installed, the answer is yes)?

**Comment by zhmstar0310** [comment]
I can get my location quickly in map APP such as Baidu Map, but I can't get it in breezy, it's very strange.
I installed google play, will this help this?
PS: My device is LG V50, I deleted lg weather push server, I think this won't affect breezy, right?

**Reply by papjul** [reply]
I didn't know you could install Play Store without installing the full Google Play Services (which requires a full clean of your system).

I think that if you can get it to work in other apps, it should work in Breezy Weather.

Once it works in your maps application, can you try refreshing in Breezy Weather? Does it work then?

**Reply by zhmstar0310** [reply]
I tried to refreshing Breezy Weather when my map APP get my locaiton, but Breezy still couldn't get my current location.
I installed Xiaomi Weather, it can get my current location normally.
Maybe I should try to reinstall Breezy and try again.

**Reply by papjul** [reply]
It's curious it only happens with this app. I will leave the community find a solution.

If you find the solution by yourself, please leave a message to help other users in the same situation.
